hazelcast/src/main/java/com/hazelcast/instance/impl/MobyNames.java

```diff
@@ -17,16 +17,16 @@
 
 package com.hazelcast.instance.impl;
 
-import com.hazelcast.internal.util.RandomPicker;
-
 import java.util.Arrays;
+import java.util.Collections;
 
 /**
  * Java port of the Moby Project random name generator (https://github.com/moby/moby).
  */
 final class MobyNames {
 
     private static final String NAME_FORMAT = "%s_%s";
+    private static final int LARGE_PRIME = 1_000_000_007;
```

Comment:
I can't see where this constant is used.

Reply:
Yep, this is a leftover from a different method of combining the adjectives and surnames that I dropped.